desktop/core/src/desktop/js/reactComponents/AlertComponent/AlertComponent.scss

```diff
@@ -0,0 +1,84 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../components/styles/variables';
+
+.flash-messages {
+    position: fixed;
+    width: 30%;
+    z-index: 9999;
+    top: 7px;
+    right: 0;
+    margin-right: 24px;
+    display: flex;
+    flex-flow: column;
+    row-gap: 10px;
+
+    button:hover,
+    button:focus {
+        background-color: unset;
+    }
+
+    button span {
+        font-size: 12px;
```

Comment:
There should be variables for font sizes, spacing etc, try an replace these hardcoded values with their variable counterparts.

Reply:
<img width="1020" alt="Screenshot 2023-10-27 at 12 15 12 PM" src="https://github.com/cloudera/hue/assets/68558847/9cf85324-efc9-4bcc-abcd-ae39e0f008a9">

Do we want to replace it with something like this?